other/appfilter.xml

```diff
@@ -6518,6 +6518,9 @@
 	<!-- Digiturk Play -->
 	<item component="ComponentInfo{com.digiturkwebtv.andstb/com.digiturkwebtv.stb.MainActivity}" drawable="digiturk_play"/>
 
+	<!-- Digiturk Play Yurtdışı-->
```

Comment:
Nice, no need to add an extra name then, right?

Reply:
Yeap, icons and files are same.
They only need appliter input.

Reply:
![image](https://github.com/Donnnno/Arcticons/assets/31142286/7bda0149-9287-4646-92cc-cc17678997b4)

I actually meant doing like this haha ^